modules/_labs/panel/react/lib/PanelHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {colors, spacing, H4, CanvasColor} from '@workday/canvas-kit-react-core';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+import {xIcon} from '@workday/canvas-system-icons-web';
+
+export interface PanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  headerTitle?: string;
+  handleClose?: () => void;
+  iconLabel: string;
+  icon: CanvasSystemIcon;
```

Comment:
What are other icon use cases? How do they close/dismiss if it's not an X?

Reply:
I think I was trying to account for another icon/link up there, but I should probably keep it simple and provide the `xIcon` and if we need to iterate over then we can consider adding this capability!